### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by Ladu-Lumori**

### CODE REVIEW

File: apps/api/package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "prisma-reset": "npx prisma migrate reset",
     "prisma-status": "npx prisma migrate status",
     "prisma-sync": "npx prisma db push",
-    "pscale-connect": "pscale connect biti dev --port 3309"
+    "pscale-connect": "pscale connect biti dev --port 3309",
+    "prisma-init": "npx prisma init",
+    "prisma-generate": "npx prisma generate"
   },
```
Comment (Emmanuel-Melon) [inline apps/api/package.json:16]: This PR containts two lock files, namely `package-lock.json` and `yarn.lock`. It's advisable to have only one lock file per project. I recommend deleting `package-lock.json`.

File: README.md [hunk 2]
```diff
@@ -114,4 +114,21 @@ export NVM_DIR="$HOME/.nvm" &&
 nvm install --lts &&
 nvm use --lts
 
-'
\ No newline at end of file
+'
+
```
Comment (Emmanuel-Melon) [inline README.md:118]: Format this content

File: apps/api/package.json (hunk 1 above)
Comment (Ladu-Lumori) [reply apps/api/package.json:16]: alright working on this, update will be seen in my next update.

Comment (Emmanuel-Melon) [reply apps/api/package.json:16]: @Ladu-Lumori resolve the conversation after you're done.

**Review by Emmanuel-Melon — APPROVED** [review]
LGTM

**Pull request merged**